scripts/cli_translations.py

```diff
@@ -18,136 +18,134 @@
-        func=translations_watch,
-        params=[Command.ParamCombination(optional_params='--verbose')],
+    # list_out_of_sync subcommand
+    out_of_sync_parser = subparsers.add_parser(
+        "list_out_of_sync",
+        help="List all translations in i18n.js files tagged as out of sync.",
     )
+    out_of_sync_parser.set_defaults(func=translations_list_out_of_sync)
 
-    Command.register_command(
-        name='export',
-        description='Export all I18N translations to a csv',
-        func=translations_export,
-        params=[
-            Command.ParamCombination(
-                required_params=('--locale', '--out'),
-                optional_params=('--missing', '--out_of_sync'),
-                description='Export all I18N translations for a given locale '
-                'to a csv specified in --out',
-            )
-        ],
+    # import subcommand
+    import_parser = subparsers.add_parser(
+        "import",
+        help="Import translations CSV into app. Required columns: filename, id, translation.",
     )
-
-    Command.register_command(
-        name='list_out_of_sync',
-        description='List all translations in i18n.js files tagged as out of sync',
-        func=translations_list_out_of_sync,
+    import_parser.add_argument(
+        "--locale",
+        type=str,
+        required=True,
+        help="Locale into which to import new translations",
     )
-
-    Command.register_command(
-        name='import',
-        description='Import translations CSV into app. Required columns: filename, id, translation',
-        func=translations_import,
-        params=[
-            Command.ParamCombination(
-                required_params=('--locale', '--input_file'),
-                description='Import new I18N translations for a given locale '
-                'from the csv specified in --input_file',
-            )
-        ],
+    import_parser.add_argument(
+        "--input_file",
+        type=str,
+        required=True,
+        help="Filename with translated text to upload to app",
     )
+    import_parser.set_defaults(func=translations_import)
 
-    Command.register_command(
-        name='add_locale',
-        description='Add a new locale to the project',
-        func=translations_add_locale,
-        params=[
-            Command.ParamCombination(
-                required_params=('--locale'), description='ISO code for new locale'
-            )
-        ],
+    # add_locale subcommand
+    add_locale_parser = subparsers.add_parser(
+        "add_locale",
+        help="Add a new locale to the project.",
+    )
+    add_locale_parser.add_argument(
+        "--locale",
+        type=str,
+        required=True,
+        help="IDO code for new locale",
     )
+    add_locale_parser.set_defaults(func=translations_add_locale)
 
-    Command.initialize_commands()
-    Command.run(Flags.ARGS.command)
+    args = parser.parse_args()
+    print(args)
```

Comment:
This prints out `Namespace(subcommand=None)` - I'm guessing this was you debugging something?